packages/db/src/scripts/import-bdb.ts

```diff
@@ -0,0 +1,49 @@
+import { parseLexicon } from './helpers/parse-lexicon';
+import { PrismaClient } from '@prisma/client';
+
+const client = new PrismaClient();
+
+const importBdb = async () => {
+  const parsed = await parseLexicon('data/lexicon/hebrew.txt', ['BdbMedDef']);
+  console.log(`Parsed ${Object.keys(parsed).length} words`);
+  await client.lemmaResource.deleteMany({
+    where: {
+      resourceCode: 'BDB',
+    },
+  });
+  const lemmaUpserts: any[] = [];
+  const data = Object.keys(parsed)
+    .filter((lemmaId) => {
+      if (typeof parsed[lemmaId]['BdbMedDef'] === 'undefined') {
+        console.error('Missing definition for', lemmaId);
+        return false;
+      }
+      return true;
+    })
+    .map((lemmaId) => {
+      // We have to create non-existent lemmas, so that the foreign key on lemma
+      // resources has something to point to.
+      lemmaUpserts.push(
+        client.lemma.upsert({
```

Comment:
You could do a create many on lemma and set `skipDuplicates: true` to just insert what was missing